.github/workflows/update.yml

```diff
@@ -0,0 +1,78 @@
+name: Update
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:main
+        uses: actions/checkout@v4
+        with: { ref: main }
+      - name: Refresh l10n:update
+        run: git push origin HEAD:update || true
+  branches:
+    needs: prepare
+    strategy:
+      max-parallel: 1
+      matrix:
+        ref: [master, beta, release, esr115]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:update
+        uses: actions/checkout@v4
+        with: { ref: update, path: l10n }
+      - name: Checkout firefox:${{ matrix.ref }}
+        uses: actions/checkout@v4
+        with:
+          repository: mozilla/gecko-dev
```

Comment:
I thought this was inefficient, as we could switch branch on a clone instead of cloning 4 times.

But it looks like each clone takes less than 5s, and for the other approach you would need to do a full clone. So, just noting it here (and probably worth documenting).

Reply:
Yeah, a shallow clone is _much_ faster esp. on a repo like mozilla-central. GitHub also hosts the runners really close to the repos, so checkouts are fast.